### PR TITLE
Use Qt-enhanced RenderParameters object for UI

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: apps/livreGUI/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,12 @@ if(NOT QT5WIDGETS_FOUND)
   return()
 endif()
 
+include(zerobufGenerateCxx)
+zerobuf_generate_qobject(ZEROBUF_GENERATED ${CMAKE_CURRENT_BINARY_DIR}
+  ${PROJECT_SOURCE_DIR}/livre/lib/zeroeq/volumeRendererParameters.fbs)
```
Comment (rdumusc) [inline apps/livreGUI/CMakeLists.txt]: There was already a zerobuf::VolumeRendererParameters, so shouldn't the generation be removed somewhere else?

Comment (tribal-tec) [inline apps/livreGUI/CMakeLists.txt]: In livre/lib it generates a Qt-less object, as no Qt dependency used and needed in livre/lib.

Comment (rdumusc) [inline apps/livreGUI/CMakeLists.txt]: Ah ok I see!

**Comment by rdumusc** [comment]
+1

**Pull request closed without merging**